src/linear_algebra/matrix/basis.lean

```diff
@@ -101,6 +101,17 @@ e.to_matrix_units_smul _
 @[simp] lemma sum_to_matrix_smul_self [fintype ι] : ∑ (i : ι), e.to_matrix v i j • e i = v j :=
 by simp_rw [e.to_matrix_apply, e.sum_repr]
 
+lemma to_matrix_vec_mul {S : Type*} [comm_ring S] [algebra R S] [fintype ι'] (b : basis ι R S)
```

Comment:
I'm not sure about this lemma.

For one thing, the reindexing is independent of the rest of the statement. I.e., if we instead proved:
```lean
lemma foo {S : Type*} [comm_ring S] [algebra R S] [fintype ι'] (b : basis ι R S)
  (v : ι' → S) [fintype ι] :
  ((b.to_matrix v).map (algebra_map R S)).vec_mul b = v :=
-- Same proof
```
then the stated result is just `foo (b.reindex f) v`.